assets/js/modules/thank-with-google/components/common/PositionRadio.js

```diff
@@ -87,36 +87,34 @@ export default function PositionRadio() {
 				</p>
 				<div className="googlesitekit-twg-position-radio__suboptions">
 					<Radio
-						id={ `button-placement-${ BUTTON_PLACEMENT_STATIC_ABOVE_CONTENT }` }
+						id={ `button-placement-${ CTA_PLACEMENT_STATIC_ABOVE_CONTENT }` }
```

Comment:
Here also for the `id` and `name` props for consistency.

Reply:
Updated.